### PR TITLE
[Issue#5779][docs]Reorganize "tiered storage" content

### LINKED ISSUE #5779: [docs] Reorganize "Tiered storage" content
**Describe the bug**
There are two sections about "tiered storage" under "Concepts and Architecture" and "Cookbooks", the information splits. It's better to reorganize the the content for "tierage storage" after merging https://github.com/apache/pulsar/pull/5762

### PR DESCRIPTION
**Pull request by atlantic2099**
Fixes #5779 
### Motivation

To reorganize "tiered storage" content to make it logically smooth.

### Modifications

Refine "tiered storage" under concepts and architecture, create new chapter for tiered storage by reorganizing content under cookbooks.

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
@atlantic2099 Thank you very much for your contributions. Except those modification, you also need to update the `../site2/website/sidebars.json` file.

**Comment by Jennifer88huang** [comment]
@atlantic2099 Good job.
When you add a new section for "Tiered storage", remove the file in line113 in the `sidebar.json` file as well.
image

File: site2/website/sidebars.json [hunk 1]
```diff
@@ -120,6 +120,12 @@
       "cookbooks-message-queue",
       "cookbooks-bookkeepermetadata"
     ],
+    "Tiered Storage": [
+      "tiered-storage-overview",
+      "tiered-principle-of-offloaders",
+      "tiered-how-to-manage-offloaders",
+      "tiered-how-to-use-offloaders"
```
Comment (Anonymitaet) [inline site2/website/sidebars.json]: A tiny suggestion for `id` in `sidebars.json`: give it a short name for easier maintenance.

**Comment by Jennifer88huang** [comment]
@atlantic2099 Could you please check my comment above on sidebars.json? It's to remove the `cookbooks-tiered-storage` file under Cookbooks, rather than `concepts-tiered-storage` under Concepts and Architecture.

**Comment by Anonymitaet** [comment]
@atlantic2099 any progress for updating this PR?

**Comment by Leon-peng2012** [comment]
Test result from Leon:
"Concepts and Architecture",
      "Tiered Storage",
           "Overview",
           "Working Principles of Offloaders",
           "Managing Offloaders",
           "Using Offloaders"

**Pull request closed without merging**